### ISSUE TITLE
Version 1.6.0 seems to be broken

### ISSUE THREAD

**Issue by betaprinz** [urbanware-org/wideredist#29]
Version 1.6.0 seems to be broken.

Maybe the bash version is the problem.

With "bash version 4.2.46(2)-release / CentOS7" wideredist 1.6.0 works fine.
with "bash 5.1.4(1)-release / Debian11" wideredist 1.6.0 seems to be broken (Download failed).

Last working verion on bash 5.1.4(1)-release is widededist 1.5.3.

sorry my bad english.

**Comment by urbanware-org** [comment]
Thank you for your feedback!

> Version 1.6.0 seems to be broken.
>
> Maybe the bash version is the problem.

I just tested the server-side script on *Red Hat* derivates:
- *CentOS* 7 with version 4.2.46(2)
- *Fedora* 36 with version 5.1.16(1)

Both work properly.

Can you check the log files, what the reason might be? 

```bash
sudo grep wideredist /var/log/syslog
```

Thank you in advance!

**Comment by urbanware-org** [comment]
Thank you!

```
wideredist160.sh: line 201: file: command not found
```

Obviously the tool `file` does not seem to be installed, which is odd.

What happens if you execute `file --help`? Do you get an error?

**Comment by betaprinz** [comment]
I installed file tool. 
Problem solved.
Thank you.

**Comment by urbanware-org** [comment]
Glad to help!

I just modified the code so the absence of the `file` command does not cause issues anymore. This will be fixed in version **1.6.1** then.
